mobile-app/app/screens/AppNavigator/screens/Loans/VaultDetail/components/VaultDetailSummary.tsx

```diff
@@ -40,7 +40,7 @@ export function VaultDetailSummary({
       <NumberRowV2
         lhs={{
           value: translate("screens/VaultDetailScreen", "Max loan amount"),
-          testID: "max_loan_amount",
+          testID: "max_loan_amount_label",
```

Comment:
is this used in any of the e2e? similarly for all the testID changes in this file

Reply:
also take note that in `NumberRowV2` it is appending `_label` to testID, so if your testID for this case would become `max_loan_amount_label_label`
<img width="680" alt="image" src="https://user-images.githubusercontent.com/40191153/198924426-a079aef7-f39d-47fd-b1e3-915541a7431b.png">

Reply:
@JoshuaC817 i believe all the newer info added in vault details v2 are not added in this e2e PR right? We can work on this in a separate PR

Reply:
noted on the `_label` will update accordingly, as for the newer info, what are the newer info that you mentioned? because now we have a new function `checkVaultDetailValues` in `loanCommands.ts` to check for the vault's status, col ratio, ID, total collateral, max loan amount, total loans, vault interest, min col ratio

Reply:
oh i think we have tests for the values and not the labels

Reply:
I will add the check for labels here